draft-ietf-add-ddr.md

```diff
@@ -391,6 +391,21 @@ resolvers that support discovery using IP addresses will need to be
 configured to present the appropriate TLS certificate when no SNI is present
 for both DoT and DoH.
 
+## Interaction with Network-Designated Resolvers
+
+Discovery of network-designated resolvers (DNR, {{?I-D.ietf-add-dnr}}) allows
+a network to provide designation of resolvers directly through DHCP {{?RFC2132}}
+{{?RFC8415}} and IPv6 Router Advertisement (RA) {{?RFC4861}} options. When such
+indications are present, clients can suppress queries for "resolver.arpa" to the
+unencrypted DNS server indicated by the network over DHCP or RAs, and the DNR
+indications SHOULD take precedence over those discovered using "resolver.arpa"
+if there is a conflict.
```

Comment:
```suggestion
for the same resolver if there is a conflict.
```

just to make it clear that DNR for resolver foo does not override DDR for resolver bar.